src/libraries/System.Private.CoreLib/src/System/ComponentModel/DefaultValueAttribute.cs

```diff
@@ -27,6 +28,7 @@ public class DefaultValueAttribute : Attribute
         /// class, converting the specified value to the specified type, and using the U.S. English
         /// culture as the translation context.
         /// </summary>
+        [PreserveDependency("ConvertFromInvariantString", "System.ComponentModel.TypeConverter", "System.ComponentModel.TypeConverter")]
```

Comment:
Can you add a todo link for https://github.com/mono/linker/issues/943? This line should not be needed once that issue is fixed.